src/views/Header.js

```diff
@@ -128,9 +128,6 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
   };
 
   _renderLeftComponent = (props: SceneProps) => {
-    if (props.scene.index === 0) {
-      return null;
-    }
```

Comment:
Can you undelete these lines and place them below `if (headerLeft)` ? That way, when no custom headerLeft implementation is given, we apply our default that doesn't render on 0 index.